README.md

```diff
@@ -17,19 +17,19 @@
 
 ---
 
-**Fast:** Yarn caches every package it downloads so it never needs to download the same package again. It also parallelizes operations to maximize resource utilization so install times are faster than ever.
+**Fast:** Yarn caches every downloaded package, so it never needs to download the same package again. It also structures operations in a parallel format to maximize resource utilization, meaning install times become even faster.
 
-**Reliable:** Using a detailed, concise lockfile format and a deterministic algorithm for installs, Yarn is able to guarantee that an install that worked on one system will work exactly the same way on any other system.
+**Reliable:** Using a detailed, concise, lockfile format and a deterministic algorithm for installments, Yarn is able to guarantee that any install that works on one system will work exactly the same on another system.
```

Comment:
Don't think `installments` conveys the right meaning here. How about `installations`?